src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintDocumentService.java

```diff
@@ -43,39 +43,52 @@ public void validateEncryptionOnUploadedDocument(CaseDocument caseDocument,
+            .binaryUrl(caseDocument.getDocumentBinaryUrl())
+            .fileName(caseDocument.getDocumentFilename())
+            .build();
+
+        byte[] pdfBytes = documentConversionService.convertDocumentToPdf(document, auth);
+        checkIfPdfIsEncrypted(errors, documentFilename, pdfBytes);
+    }
+
+    private void handlePdfFile(CaseDocument caseDocument, String auth, List<String> errors, String documentFilename, String caseId) {
+        byte[] pdfBytes = service.download(caseDocument.getDocumentBinaryUrl(), auth);
+
+        if (pdfBytes != null) {
+            checkIfPdfIsEncrypted(errors, documentFilename, pdfBytes);
+        } else {
+            String errorMessage = "Uploaded document " + documentFilename + " is empty.";
+            log.error("Uploaded document {} for Case ID: {} is empty", documentFilename, caseId);
+            errors.add(errorMessage);
+        }
+    }
 
-            if (pdfBytes != null) {
-                try (PDDocument doc = Loader.loadPDF(pdfBytes)) {
-                    if (doc.isEncrypted()) {
-                        errors.add("Uploaded document '" + documentFilename + "' contains some kind of encryption. "
-                            + "Please remove encryption before uploading or upload another document.");
-                    }
-                } catch (InvalidPasswordException ipe) {
-                    String errorMessage = "Uploaded document '" + documentFilename + "' is password protected."
-                        + " Please remove password and try uploading again.";
-                    errors.add(errorMessage);
-                    log.error(ipe.getMessage());
-                } catch (IOException exc) {
-                    String errorMessage = "Failed to parse the documents for " + documentFilename;
-                    errors.add(errorMessage + "; " + exc.getMessage());
-                    log.error(exc.getMessage());
-                }
-            } else {
-                String errorMessage = "Uploaded document " + documentFilename + " is empty.";
-                log.error("Uploaded document {} for Case ID: {} is empty", documentFilename, caseId);
-                errors.add(errorMessage);
+    private void checkIfPdfIsEncrypted(List<String> errors, String documentFilename, byte[] pdfBytes) {
+        try (PDDocument doc = Loader.loadPDF(pdfBytes)) {
+            if (doc.isEncrypted()) {
+                errors.add("Uploaded document '" + documentFilename + "' contains some kind of encryption. "
+                    + "Please remove encryption before uploading or upload another document.");
             }
+        } catch (InvalidPasswordException ipe) {
+            String errorMessage = "Uploaded document '" + documentFilename + "' is password protected."
+                + " Please remove password and try uploading again.";
+            errors.add(errorMessage);
+            log.error(ipe.getMessage());
```

Comment:
There is no need to log this error. We don't want to see this as an exception in our alerts